src/Stryker.Core/Stryker.Core/Reporters/ClearTextTreeReporter.cs

```diff
@@ -82,7 +82,8 @@ public void OnAllMutantsTested(IReadOnlyProjectComponent reportComponent, TestPr
                         _ => $"[Red][[{mutant.ResultStatus}]][/]",
                     };
 
-                    var mutantNode = fileNode.AddNode(status + $" {mutant.Mutation.DisplayName} on line {mutant.Line}");
+                    var mutantLine = mutant.Mutation.OriginalNode.GetLocation().GetLineSpan().StartLinePosition.Line + 1;
```

Comment:
Isn't it better to make the Mutation class generic and solve this in there? Because this line is still C# specific. If we add a `GetLine()` method to `Mutation` we could just get the line here, both for C# and F#.

Reply:
@richardwerkman this is the end goal. It's just damn hard to do right away :D
I mean, that's what I did back in a day but then most of the data structures had to become double-generic and like 50% of the code base changed. 

But eventually we'll get there and some things will streamline. I hope.

Reply:
This whole thing feels like Jenga game - I am poking different pieces and taking whatever I can take without crashing the whole thing right away :D

Reply:
Yeah I know what you mean. We need to split stryker into smaller pieces to make changes easier! It's on our backlog ;)